src/python/bot/fuzzers/syzkaller/runner.py

```diff
@@ -95,7 +95,20 @@ def repro(self, repro_timeout, repro_args):
     logs.log('Running Syzkaller testcase.')
     additional_args = copy.copy(repro_args)
     result = self.run_and_wait(additional_args, timeout=repro_timeout)
-    logs.log('Syzkaller testcase stopped.')
+    reproducible = False
+    if 'all done.' in result.output:
+      regex = re.compile(r'reproduced (\d+) crashes')
```

Comment:
compile this once and save this as a class constant.

Reply:
+1, class constant compiled var once.

Reply:
Done.